tests/docker-conf/phpfpm/profiles.ini.php

```diff
@@ -162,4 +162,8 @@
 ;db=2
 ;ttl=0
 
-
+[webdav:default]
```

Comment:
Should the distributed `profiles.ini.php.dist` be updated with a commented example ? (at least to advertise a little bit these new variables)
lizmap/var/config/profiles.ini.php.dist

If yes, I would add a quick doc and warning about webdav URL being exposed in the WFS service, so we can prevent any users complaining about this security "issue".

Adding the warning `profiles.ini.php.dist` will warn sysadmin about this.